### PR TITLE
V.0.1.0

### PR DESCRIPTION
**Pull request by StianSto**
Q: should i have used @use instead of @import? 

PS: accidentally merged into main once, has caused a few conflicts. will be fixed after review, when merging v. 0.1 into main 

this project is a SoMe app where the user has a lot of control over customizing their profile and how they appear to the world.

### CODE REVIEW

**Comment by Fermain** [comment]
Please add `.DS_Store` to your `.gitignore` file and remove instances of this file in your project.

This is a Mac OS system file that is used to store file thumbnails, so we don't need it!

**Comment by Fermain** [comment]
Please note that this branch has merge conflicts that prevent it from being merged. A merge conflict arises when the same file is edited in two different branches. Git does not know which change to honour, so it opens a conflict for manual resolution. Resolving a conflict can be quite quick and easy, please get in touch if you would like guidance here.

**Review by Fermain — COMMENTED** [review]
I absolutely love this submission and I don't often say that in a pull review.

There's a few things to address but you have taken this in the exact right direction.

File: src/scss/base/_boilerplate.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+
+body{ overflow-x: hidden; }
```
[the remark this answers is not in the document]
Comment (StianSto) [reply src/scss/base/_boilerplate.scss]: no you are right it's not needed. sidebar has fixed widths. it has been removed 👍

File: src/scss/components/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+//like
+.btn__like {
+    & .liked {
+        color: rgb(224, 57, 38);
```
[the remark this answers is not in the document]
Comment (StianSto) [reply src/scss/components/_buttons.scss]: you're right. rgb value added as the new $red in variables. might add a $like variable to theme in the next version. .liked color changed to $red.

File: src/scss/components/_menu-bar.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// menu icon nav bar
+$gap-bars: 25%;
```
[the remark this answers is not in the document]
Comment (StianSto) [reply src/scss/components/_menu-bar.scss]: yup, you're right. both $gap-bars and $height-bars have been moved to _variables.scss

**Pull request merged**